packages/nimble-components/src/wafer-map/modules/zoom-handler.ts

```diff
@@ -6,8 +6,11 @@ import {
     ZoomTransform,
     zoomTransform
 } from 'd3-zoom';
-import type { DataManager } from './data-manager';
-import type { RenderingModule } from './rendering';
+import type { WaferMap } from '..';
+
+export interface ZoomEvent {
```

Comment:
The `ZoomEvent` interface might not need to be exported since it appears to only be used within this file.